src/EmergencyContactForm/EmergencyContactForm.js

```diff
@@ -1,11 +1,43 @@
 import React from 'react'
+// import React, { useState } from 'react';
+import './EmergencyContactForm.css'
 
-const EmergencyContactForm = () => {
+class EmergencyContactForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '', 
+      phone: '', 
+      relationship: ''
+    }; 
+    this.formUpdate = this.formUpdate.bind(this);
+  }
+
+  formUpdate = (event) => {
+    this.setState({[event.target.name]: event.target.value});
+  }
+
```

Comment:
This looks great! I know that establishing state and updating it can be tricky, so lets tinker more with it tomorrow. Nice work :)